vector/src/main/codegen/templates/BaseWriter.java

```diff
@@ -101,6 +103,13 @@ public interface MapWriter extends ListWriter {
     MapWriter value();
   }
 
+  public interface ExtensionWriter extends BaseWriter {
+    void writeNull();
+    <T extends ExtensionHolder> void write(T var1);
```

Comment:
Same here - why not just `void write(ExtensionHolder value)`?

Reply:
fixed